### ISSUE TITLE
when I try to send message not go and the verification sms not work

### ISSUE THREAD

**Issue by jambolo1970** [loqui/im#1223]
I write the message, and when i tap to send it, it's not go,
on firefoxOS 1.3
with Loqui v0.5.16

and the verification sms not work, i have access thanks a old version, and i insert number of code 123456, then i have access to the new version 5.16, but i can't send any massage as https://github.com/loqui/im/issues/1209 It's the same problem.

**Comment by nfsprodriver** [comment]
Tanks for report! You seem to have built your installation from latest commit. Currently the repo is BROKEN so it'll be fixed as soon as possible. I also had problems with SMS verification. Call method worked.

**Comment by jambolo1970** [comment]
yes i build it on my PC linux OpenSuse 42.2 , and i attach the how to build that I followed
[installazione_im.zip](https://github.com/loqui/im/files/890840/installazione_im.zip)
it did not give me errors in the compilation

**Comment by jambolo1970** [comment]
the problem on sms
[IM.zip](https://github.com/loqui/im/files/890851/IM.zip)
I hope you helpful, because i like this app

**Comment by nfsprodriver** [comment]
You must not enter the country again. Anyway I also had problems with SMS. If this doesn't work, please try voice call.
